spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -349,6 +349,7 @@
             user_id: anything,
             request_ip: request.remote_ip,
             ipp_enrollment_in_progress: false,
+            proofing_components: Idv::ProofingComponents,
```

Comment:
What is the effect of passing in a class name here rather than a discrete value? Is it effectively asserting that it's an instance/type of that class?

Reply:
Yes, [internally](https://github.com/rspec/rspec/blob/a8d1f214883b1d1f2b1031f95dac36690405e835/rspec-support/lib/rspec/support/fuzzy_matcher.rb#L20) rspec is doing `Idv::ProofingComponents === proofing_components`